openfisca_aotearoa/entities.py

```diff
@@ -3,38 +3,34 @@
 # This file defines the entities needed by our legislation.
 from openfisca_core.entities import build_entity
 
-Household = build_entity(
-    key = "household",
-    plural = "households",
-    label = u'Household',
-    doc = '''
-    Household is an example of a group entity.
-    A group entity contains one or more individual·s.
-    Each individual in a group entity has a role (e.g. parent or children). Some roles can only be held by a limited number of individuals (e.g. a 'first_parent' can only be held by one individual), while others can have an unlimited number of individuals (e.g. 'children').
 
-    Example:
-    Housing variables (e.g. housing_tax') are usually defined for a group entity such as 'Household'.
+# Using middle English spelling due to `property` being a Python keyword.
+Propertee = build_entity(
```

Comment:
I'm a bit worried that this name will not be obvious to remember throughout the code, especially since the label and the varname are different.

Could it be worth investigating a bit which [synonyms](http://www.thesaurus.com/browse/property?s=t), rather than homophones, are available for `property`? 🙂 

For example, I quite like `estate` or even `house`. 🙂

Reply:
I will leave this in for this pull request and instigate a wider discussion.